run-tests.php

```diff
@@ -87,13 +87,9 @@ function show_usage()
     --help
     -h          This Help.
 
-    --html <file> Generate HTML output.
-
     --temp-source <sdir>  --temp-target <tdir> [--temp-urlbase <url>]
                 Write temporary files to <tdir> by replacing <sdir> from the
-                filenames to generate with <tdir>. If --html is being used and
-                <url> given then the generated links are relative and prefixed
-                with the given url. In general you want to make <sdir> the path
+                filenames to generate with <tdir>. In general you want to make <sdir> the path
```

Comment:
There's probably some line length limit (80?) for the usage output.

Reply:
resolved.